index.html

```diff
@@ -3283,14 +3284,14 @@ <h3>Refreshing</h3>
           title="Usage of the refreshService property by an issuer">
 {
   "@context": [
-    "https://www.w3.org/2018/credentials/v1",
-    "https://w3id.org/age/v1",
-    "https://w3id.org/security/suites/ed25519-2020/v1"
+    "https://www.w3.org/ns/credentials/v2",
+    "https://www.w3.org/ns/credentials/examples/v2",
+    "https://w3id.org/age/v1"
```

Comment:
```suggestion
    "https://www.w3.org/2018/credentials/v1",
    "https://w3id.org/age/v1",
    "https://w3id.org/security/suites/ed25519-2020/v1"
```

This is a real production credential (from TruAge); we shouldn't change it (because it's shipped in production) and because the WG said that we'd use "real" examples.